odoo/tests/common.py

```diff
@@ -718,6 +719,19 @@ def setUpClass(cls):
 
         cls.cr = cls.registry.cursor()
         cls.addClassCleanup(cls.cr.close)
+        if cls.rollback_sequences:
+            cls.cr.execute("SELECT sequencename, last_value FROM pg_sequences")
+            skip_sequences = {'ir_profile_id_seq', 'ir_sequence_id_seq', 'bus_bus_id_seq'}
```

Comment:
Any suggestion for the name of this classlevel constant? :)

Reply:
"non rollbacked sequences"? 

Not sure about classlevel though that means this bit here has to check through the entire MRO to accumulate the set, or users of the feature have to remember to copy the base set. And this seems like a model-level property rather than a test-level one.